pkg/ui/workspaces/db-console/src/views/shared/components/summaryBar/summarybar.styl

```diff
@@ -100,6 +100,9 @@
   &--number &__value
     color $colors--primary-green-3
 
+  &--alert-unavailable-ranges  &__value
```

Comment:
Sorry, this comment got lost in my original review. 

Can we change this class name so it's not specific to unavailable ranges? Maybe something like `number-alert`?

Reply:
> Sorry, this comment got lost in my original review.
> 
> Can we change this class name so it's not specific to unavailable ranges? Maybe something like `number-alert`?

Hi @abarganier Thanks for suggestion,
I have updated the code.